portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,44 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400&display=swap" rel="stylesheet">
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div class="left">
+            <img src="images/profile-pic.jpg" id="profile-pic">
```

Comment:
Google's HTML style guide recommends indenting with 2 spaces, and it looks like you used 4 here.

https://google.github.io/styleguide/htmlcssguide.html#Indentation

I don't want to be super strict about minor styling issues, so it's up to you whether you want to follow these recommendations or not.